tools/install-powershell.ps1

```diff
@@ -79,6 +79,65 @@ Function Remove-Destination([string] $Destination) {
     }
 }
 
+Function Test-PathInRegistryPath([string] $Path) {
+    #Remove ending DirectorySeparatorChar for comparison purposes
+    $Path = [System.Environment]::ExpandEnvironmentVariables([regex]::Replace($Path, '\\;$|\\$', ''));
+    $InstalledPaths = @()
+    #[System.Environment]::GetEnvironmentVariable automatically expands all variables
```

Comment:
NIT: convention is to have a newline before a comment
(NIT = minor issues not blocking the PR)

Reply:
blocks of comments just have one newline  before the first comment